airflow/executors/base_executor.py

```diff
@@ -56,6 +56,8 @@ class BaseExecutor(LoggingMixin):
         ``0`` for infinity
     """
 
+    _job_id: Optional[str] = None
```

Comment:
```suggestion
    job_id: Optional[str] = None
    """job_id of the SchedulerJob this executor is running under"""
```

Reply:
@ashb this wouldn't work. you can't have a public member with the same name as a property it creates an infinite loop.